### PR TITLE
Make meshes faster

### PR DESCRIPTION
**Pull request by asinghvi17**

### CODE REVIEW

**Comment by asinghvi17** [comment]
Libfreetype not defined?!  @simon

**Comment by SimonDanisch** [comment]
I guess its that: https://github.com/JuliaGraphics/FreeTypeAbstraction.jl/issues/36

File: Project.toml [hunk 1]
```diff
@@ -12,7 +12,7 @@ FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
 GeometryTypes = "4d00f742-c7ba-57c2-abde-4428a4b178cb"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
-FreeType = "663a7486-cb36-511b-a19d-713bb74d65c9"
+FreeType = "b38be410-82b0-50bf-ab77-7b57e271db43"
```
Comment (SimonDanisch) [inline Project.toml]: Really? The uuid was wrong? :D

Comment (asinghvi17) [reply Project.toml]: yeah, I copied from FTA instead of FT

**Comment by asinghvi17** [comment]
font handling is borked:
> LoadError: MethodError: no method matching getindex(::FreeTypeAbstraction.FTFont, ::Int64)

**Comment by asinghvi17** [comment]
K...this seems to work as far as I can tell, but I don't know how good it is.  Will experiment with some GeoMakie stuff.

**Comment by SimonDanisch** [comment]
lol we get rate limited from github...well I count this as a tests passing^^

**Pull request merged**